go-controller/pkg/util/pod_annotation.go

```diff
@@ -571,6 +592,19 @@ func AddRoutesGatewayIP(
 					podAnnotation.Gateways = append(podAnnotation.Gateways, gatewayIPnet.IP)
 				}
 			}
+			// Until https://github.com/ovn-kubernetes/ovn-kubernetes/issues/4876 is fixed, it is limited to IC only
+			if config.OVNKubernetesFeature.EnableInterconnect {
+				if _, isIPv6Mode := netinfo.IPMode(); isIPv6Mode {
+					joinAddrs, err := ParseNodeGatewayRouterJoinAddrs(node, netinfo.GetNetworkName())
+					if err != nil {
+						if IsAnnotationNotSetError(err) {
+							return types.NewSuppressedError(err)
+						}
+						return fmt.Errorf("failed parsing node gateway router join addresses, %w", err)
```

Comment:
```suggestion
						return fmt.Errorf("failed parsing node gateway router join addresses, %w", err)
```
for which network would be helpful message to include in this error...

Reply:
done